iphone/Classes/TiUIAttributedStringProxy.m

```diff
@@ -178,11 +214,7 @@ -(void)addAttribute:(id)args
             break;
             
         case AttributeNameLineBreak:
-            attrName = NSParagraphStyleAttributeName;
-            NSMutableParagraphStyle *paragraphStyle = [[[NSMutableParagraphStyle alloc] init] autorelease];
-            NSNumber * num = [TiUtils numberFromObject:value];
-            [paragraphStyle setLineBreakMode:[num unsignedIntegerValue]];
-            attrValue = paragraphStyle;
+            DebugLog(@"[WARN] Ti.UI.AttributeNameLineBreak is deprecated, you must use AttributeNameParagraphStyle.lineBreak");
```

Comment:
Use the `DEPRECATED_REPLACED` macro instead. This could work:
```objc
DEPRECATED_REPLACED(@"UI.ATTRIBUTE_LINE_BREAK", @"UI.ATTRIBUTE_PARAGRAPH_STYLE.linebreak", @"6.2.0");
```
Basically, this will generate a unified warning for you. The "Ti" is not needed, since it will be processed by the macro definition. Bu